test_client/src/main/java/io/reactivesocket/tck/TestSubscriber.java

```diff
@@ -201,6 +202,7 @@ public void onNext(T t) {
         Payload p = (Payload) t;
```

Comment:
Just noticed the TestSubscriber, it's rather involved – are we sure it generally speaking abides the RS spec?
Can we stress it using the RS TCK or is it too weird? Looks like a test probe I guess.